src/QMCHamiltonians/CoulombPBCAB.cpp

```diff
@@ -569,6 +625,26 @@ void CoulombPBCAB::initBreakup(ParticleSet& P)
     Vat.resize(NptclA,V0);
     Vspec.resize(NumSpeciesA,0);//prepare for PP to overwrite it
   }
+  
+  //If ComputeForces is true, then we allocate space for the radial derivative functors.
+  if(ComputeForces)
+  {
+    dAB = LRCoulombSingleton::getDerivHandler(P);
+    if(fV0==0)
+      fV0=LRCoulombSingleton::createSpline4RbyVs(dAB,myRcut,myGrid);
+    if(dfV0==0)
+      dfV0=LRCoulombSingleton::createSpline4RbyVsDeriv(dAB,myRcut,myGrid);
+    if(fVat.size())
+    {
+      app_log() << "  fVat is not empty.  Something is wrong" << std::endl;
+      OHMMS::Controller->abort();
```

Comment:
Use APP_ABORT. If a non root rank hit here. Root doesn't have a chance to print.

Reply:
Done.